apps/design/backend/src/features.ts

```diff
@@ -134,6 +134,22 @@ export const userFeatureConfigs = {
     BALLOT_LANGUAGE_CONFIG: true,
   },
 
+  demos: {
+    ACCESS_ALL_ORGS: false,
+    SYSTEM_SETTINGS_SCREEN: true,
+    MARGINAL_MARK_THRESHOLD: false,
+    EXPORT_SCREEN: true,
+    CHOOSE_BALLOT_TEMPLATE: false,
```

Comment:
Will ballot template default to VX or NH? Is that what we want? Maybe we should turn this feature on